stable/swagger/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "swagger.fullname" . }}
+  labels:
+    app: {{ template "swagger.name" . }}
+    chart: {{ template "swagger.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.externalPort }}
+      targetPort: {{ .Values.service.internalPort }}
+      protocol: {{ .Values.service.protocol }}
+      name: {{ .Values.service.name }}
```

Comment:
This is not the service name, it's the name of the port, which you'll probably want to hard-code.